src/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import CTAButton from '../CTAButton/CTAButton';
+import {
+  HeroSectionWrapper,
+  HeroSectionTitle,
+  HeroSectionP,
+  HeroSectionButtonLinkWrapper,
+  Table,
+  TableContainer,
+} from './HeroSection.styles';
+
+const HeroSection = () => (
+  <HeroSectionWrapper>
+    <HeroSectionTitle>No reactions to your reddit posts?</HeroSectionTitle>
+    <HeroSectionP>
+      Great timing, great results! Find the best time to post on your subreddit.
+    </HeroSectionP>
+    <HeroSectionButtonLinkWrapper to="/search?q=javascript">
```

Comment:
Instead of the ButtonLinkWrapper you can directly add margins etc to the `CTAButton` like

```
export const Button = styled(CTAButton)`
...
`
```

Then you make it a link using the `as` prop

```
<Button as={Link} to="...">
```

Reply:
Thank you! Didn't know that and it looks neat